extension/src/setup/index.ts

```diff
@@ -83,38 +75,23 @@ export class Setup
   private dotFolderWatcher?: Disposer
 
   constructor(
-    stopWatch: StopWatch,
     config: Config,
-    dvcExecutor: DvcExecutor,
-    dvcReader: DvcReader,
-    dvcRunner: DvcRunner,
-    gitExecutor: GitExecutor,
-    gitReader: GitReader,
```

Comment:
[F] All of these `ICli`s sit behind the `InternalCommands` facade. No point to include the functionality twice.